themes/stellaris_theme.json

```diff
@@ -239,7 +300,7 @@
       }
     },
     {
-      "scope": "entity.name.tag.yaml",
+      "scope": "entity.name.tag",
```

Comment:
Wouldn't this change make it so this color is applied to all tags, in all languages, not just yaml?

Reply:
Not sure, but I wasn't expecting this theme to be used with any other language. I use the extension "Theme by language" so I've only looked at how this theme looks in YAML. A more specific alternative could be 
"scope": [ "entity.name.tag.yaml", "entity.name.tag.stellaris.yaml" ]

Reply:
I say let's keep the `entity.name.tag.yaml` scope that we had before. We can't assume that everyone is using that extension and we should try to minimize our dependency on other extensions.